<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoodsAddDialog</class>
 <widget class="QDialog" name="GoodsAddDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>148</width>
    <height>147</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>148</width>
    <height>147</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>148</width>
    <height>147</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加商品</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>单位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="unitEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>价格</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="priceEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
